index.html

```diff
@@ -48,7 +49,7 @@
   <script src="https://cdnjs.cloudflare.com/ajax/libs/i18next-http-backend/1.3.1/i18nextHttpBackend.min.js"></script>
 
   <!-- CONFETTI JS - https://github.com/mathusummut/confetti.js -->
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/canvas-confetti/1.6.0/confetti.min.js" integrity="sha512-n4mkhsTTwnEox+TGRnxJmtvrU+Rgec7NUjrHmZE7RlYOim+XLwgMMe1/jovYl/Eh24hvPLKgX0QGdPfATAphqQ==" crossorigin="anonymous"></script>
+  <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/canvas-confetti/1.6.0/confetti.min.js" integrity="sha512-n4mkhsTTwnEox+TGRnxJmtvrU+Rgec7NUjrHmZE7RlYOim+XLwgMMe1/jovYl/Eh24hvPLKgX0QGdPfATAphqQ==" crossorigin="anonymous"></script> -->
```

Comment:
Delete instead of commenting out.

Reply:
Removed unused library imports

Reply:
The point of this PR to give a the user an ability to contain marker adjustments subsequent to an address lookup. I added a standard center circle for minor adjustments. An larger area appears around it when Nominatim is unable to made an exact or the supplied address is not local enough.  

As long Nominatim returns a valid result, a ~20 meter radius wide circle is drawn around the returned center point of the initial address lookup. The circle meant to be roughly size of a large building so the user can set the the exact location of their business without restriction.  In case the returned bounding box was larger than the standard circle  the supplied would be used to draw a polygon around it. This polygon expanded the area the marker could move around in. It also add the restriction that the marker must always be in valid place according to Nominatim. At no point will the marker be allowed outside the larger polygon. 

Whenever the marker was dropped in this secondary polygon its location is sent to Nominatim as a reverse lookup. If that lookup fails then the marker is movement was rolled back. Failures happen when the it was placed in the ocean or some other large body of water. Otherwise the marker's location must fall inside the bounding box returned by the reverse lookup. In case it doesn't, the marker is moved to the location of point returned by the same  lookup. This allows a marker dropped in a small lake  to automatically end up on the nearest shore. 

I hope this explains how this PR work. If not, I'll be happy to create a couple of animated gifs demonstrating the functionality . I never made one before so I appreciate if you include a link to a tutorial.

Reply:
I found Peek so I made two in the moment I had. Might be good to test know cases if they exist.

![near_miss_78_holy_ oak](https://user-images.githubusercontent.com/1075268/226687051-510b43b1-b28d-4494-9ecf-226c0aed52d9.gif)
![onosm_exact_search](https://user-images.githubusercontent.com/1075268/226687054-6bc46156-b32e-46b6-8972-1798eb8ba576.gif)